backend/blueprints/spa_api/service_layers/replay/predicted_ranks.py

```diff
@@ -33,13 +34,38 @@ def create_from_id(id_: str, session=None) -> List['PredictedRank']:
         accepted_playlists = [
             13,  # standard
             3,  # unranked standard
-            6,  # custom
+            6,  # custom,
```

Comment:
is it required to be in this exact order?

just thinking that putting them in numbered order might be easier to read